TLM/TLM/Manager/Impl/ExtPathManager.cs

```diff
@@ -9,9 +10,13 @@ public class ExtPathManager
         : AbstractCustomManager,
           IExtPathManager
     {
+        private readonly Vector2[] _spiralGridCoordsCache;
+
         public static readonly ExtPathManager Instance = new ExtPathManager();
 
         private ExtPathManager() {
+            var radius = 3;
+            _spiralGridCoordsCache = LoopUtil.GenerateSpiralGridCoordsClockwise(3).ToArray();
```

Comment:
Not directly, because the radius can be different for them all.
But you are right in the sense that we could make a function that calculates the largest spiral necessary, and pass that to all managers that need a spiral. That would not be a relevant perf increase though, since that is done only once during start. And it would increase code complexity a little. So its doable but I'm not sure if its worth it.